src/flutter/shell/platform/linux_embedded/surface/environment_egl.h

```diff
@@ -24,20 +24,13 @@ class EnvironmentEgl {
       return;
     }
 
-    if (eglInitialize(display_, nullptr, nullptr) != EGL_TRUE) {
```

Comment:
I replaced `template T` with `EGLNativeDisplayType`. Also, I didn't create `environment_egl.cc` file because it requires consideration of the include order of `gbm.h`.

Reply:
I've requested you to replace it to remove the following template <D>. Template <D> is not used. So, please remove it.

context_egl.h
```
template <typename D, typename W>
class ContextEgl {
 public:
```

Reply:
I fixed it.